src/sync/SyncQueue.js

```diff
@@ -74,6 +82,7 @@ export class SyncQueue {
           const existingSyncOutRecord = this.database
             .objects('SyncOut')
             .filtered('recordId == $0', record.id)[0];
+          if (!existingSyncOutRecord && recordType === 'SensorLog' && changeType === DELETE) return;
```

Comment:
should only stop the deletion of !existing sensor logs syncing (there was something i was trying to protect against, something that could break when upgrading original version of VM mobile apk). Most likely this can be deleted in the near future once every site is on this version